src/features/campaigns/components/CampaignDetailsForm.tsx

```diff
@@ -58,71 +52,64 @@ const CampaignDetailsForm = ({
   };
 
   return (
-    <Form
-      initialValues={initialValues}
+    <Box
+      component="form"
```

Comment:
omg i never thought about doing it this way! I've only used a html `form` tag with a MUI `Box` inside for styling. Cool beans 🫘